docs/tutorials/WritingBlocksToFile.md

```diff
@@ -7,137 +7,62 @@ useful for both building and debugging conduit pipelines.
-  block-dir: "/path/to/block/files"
-  # FilenamePattern is the format used to write block files. It uses go
-  # string formatting and should accept one number for the round.
-  # If the file has a '.gz' extension, blocks will be gzipped.
-  # Default: "%[1]d_block.json"
-  filename-pattern: "%[1]d_block.json"
-  # DropCertificate is used to remove the vote certificate from the block data before writing files.
-  drop-certificate: true
-```
-
 ## Setting Up Our Pipeline
 
-Let's start assembling a configuration file which describes our conduit pipeline. For that we'll run
+Let's start assembling a configuration file which describes our conduit pipeline. For that we'll use the `conduit init` command to create the `conduit.yml` template.
 
 ```bash
-conduit init -d data
+mkdir data
+conduit init --importer algod --processor filter_processor --exporter ... > data/conduit.yml data
 ```
 
-This will create a configuration directory if we don't provide one to it, and write a skeleton config file
-there which we will use as the starting point for our pipeline. Here is the config file which the `init` subcommand has
-written for us:
-
-```yaml
-# Generated conduit configuration file.
-log-level: INFO
-# When enabled prometheus metrics are available on '/metrics'
-metrics:
-  mode: OFF
-  addr: ":9999"
-  prefix: "conduit"
-# The importer is typically an algod archival instance.
-importer:
-  name: algod
-  config:
-    netaddr: "your algod address here"
-    token: "your algod token here"
-# One or more processors may be defined to manipulate what data
-# reaches the exporter.
-processors:
-# An exporter is defined to do something with the data.
-# Here the filewriter is defined which writes the raw block
-# data to files.
-exporter:
-  name: file_writer
-  config:
-  # optionally provide a different directory to store blocks.
-  #block-dir: "path where data should be written"
-```
+This will create a configuration directory and write the `conduit.yml` config file
+template.
 
 ## Setting up our Importer
 
-We can see the specific set of plugins defined for our pipeline--an `algod` importer and `file_writer` exporter.
-Now we will fill in the proper fields for these. I've got a local instance of algod running at `127.0.0.1:8080`,
-with an API token of `e36c01fc77e490f23e61899c0c22c6390d0fff1443af2c95d056dc5ce4e61302`. If you need help setting up
+Now we will fill in the algod importer configuration. I've got a local instance of algod running at `127.0.0.1:8080`,
+with an API token of `e36c01fc77e490f23e61899c0c22c6390d0fff1443af2c95d056dc5ce4e61302` and aan admin API token of `2d9ceaf47debb8aff77c62475b8337f6ecef75c2aa8aa02170a8d38b9126b75a`. Find these in the `algod.token` and `algod.admin.token` files from your algod data directory. If you need help setting up
 algod, you can take a look at the [go-algorand docs](https://github.com/algorand/go-algorand#getting-started) or our
 [developer portal](https://developer.algorand.org/).
```

Comment:
Hmmm I'm just thinking about how to help people get algod set up properly for conduit. In the other tutorial (Indexer Writer one) we do just walk them through how to do it. I wonder if that snippet could be included in the algod importer plugin documentation page, and then we could point to it from anywhere? Because there are specifics for the importer (in particular the follow mode) that people need to know. Here they wouldn't get that information.

Reply:
Agreed, this link is not very useful to someone new.